src/lib/airtable/schema.js

```diff
@@ -1,6 +1,6 @@
 /*
-    THIS IS A GENERATED FILE
```

Comment:
Can you revert this file? Your PR shouldn't have changes to `request.js` or `schema.js`